.github/workflows/bootzooka-ci.yml

```diff
@@ -39,6 +39,10 @@ jobs:
             ~/.coursier
           key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.sbt') }}
 
+      - name: Generate OpenAPI Spec
+        id: generate-openapi-spec
+        run: sbt "backend/generateOpenAPISpec"
+
```

Comment:
before running tests, we need an `openapi.yaml` file, which is then used to generate proper types (otherwise, tests would fail)